pkg/models/system.go

```diff
@@ -0,0 +1,18 @@
+package models
+
+import "github.com/google/uuid"
+
+// System is the model for a system
+type System struct {
```

Comment:
We should have a `SystemShort` (or `SystemOverview`, or someone think of a better name, please) as discussed during pairing, I think... 
You should be able to do something like this:
```
type SystemShort struct {
  ID uuid.UUID
  Acronym string
  Name string
}

type System struct {
  SystemShort
  Description string
  //...
}
```

Reply:
Feel free to do this when `/system` gets implemented.

Reply:
I split it out here but I didn't create the /system endpoint